gef.py

```diff
@@ -7663,8 +7663,11 @@ def context_source(self) -> None:
 
         nb_line = self["nb_lines_code"]
         fn = symtab.filename
-        if len(fn) > 20:
-            fn = f"{fn[:15]}[...]{os.path.splitext(fn)[1]}"
+        if len(fn) > 30:
```

Comment:
Can you make those values settings for the `context` command instead? This way people can customize it according to their own setup/needs.

Reply:
I've added 3 variables and documented them.

have a look at e80d01b64b6caaf989e8cea11d3273c0a3556f12